commit b6aa92725eaf823c4b18316729eae494783daa85
Author: Olivier Houchard <ohouchard@haproxy.com>
Date:   Mon May 6 18:58:48 2019 +0200

    MINOR: doc: Document allow-0rtt on the server line.
    
    Briefly document allow-0rtt on the server line, and only the part that apply
    to 1.8 and 1.9.
    
    This should be backported to 1.8 and 1.9.
    
    (cherry picked from commit 8cb2d2e94199b8a6a9186ec12ee8146421a5d227)
    Signed-off-by: Willy Tarreau <w@1wt.eu>
    (cherry picked from commit 895b6a4568287b87d69599f347da01dcd1cfc9b2)
    [wt: context]
    Signed-off-by: Willy Tarreau <w@1wt.eu>

diff --git a/doc/configuration.txt b/doc/configuration.txt
index 7768e761..720b32e3 100644
--- a/doc/configuration.txt
+++ b/doc/configuration.txt
@@ -11223,6 +11223,10 @@ agent-port <port>
 
   See also the "agent-check" and "agent-inter" parameters.
 
+allow-0rtt
+  Allow sending early data to the server when using TLS 1.3.
+  Note that early data will be sent only if the client used early data.
+
 backup
   When "backup" is present on a server line, the server is only used in load
   balancing when all other non-backup servers are unavailable. Requests coming
